sui/p2p_transfer/index.js

```diff
@@ -19,48 +20,47 @@ const main = async () => {
   const SENDER_PRIVATE_KEY = process.env.ACC1_PRIVATE_KEY;
   const sender_keypair = getKeyPairFromExportedPrivateKey(SENDER_PRIVATE_KEY);
 
-  const RECIEVER_PRIVATE_KEY = process.env.ACC2_PRIVATE_KEY;
-  const receiver_keypair = getKeyPairFromExportedPrivateKey(RECIEVER_PRIVATE_KEY);
-  const receiver_address = receiver_keypair.getPublicKey().toSuiAddress();
-
 
   // create a new SuiClient object pointing to the network you want to use
   let url = getFullnodeUrl('mainnet');
   if (URL_OVERRIDE) {
       url = URL_OVERRIDE;
   }
   const suiClient = new SuiClient({ url: url });
+  let gasCoin = null
 
   while (true) {
```

Comment:
Works once, but fails when it loops back
We see this error:
"Error: Transaction execution failed due to issues with transaction inputs, please review the errors and try again: Object (0x<>, SequenceNumber(80724588), o#An9trEDTQi2rAVTGgaqd669fXvM7mYDDqFFYR4U1LLyd) is not available for consumption, its current version: SequenceNumber(80724595)..

Reply:
Are you concurrently running Both sui tests off of the same account?  This error implies that the gas coin is being used between runs

Reply:
Yup, we do not want the account to be exclusive for the measurement.

Reply:
That seems like an impractical restriction.  Fundamentally, a lot of the SUI SDK is designed to assume you will won't be executing from multiple apps concurrently.  If you build 2 transaction in parellel using the default options and then execute them, one is guaranteed to fail.  This doesn't seem like a realistic way to run these bench marks, but there are a number of ways we can make this work

* Run both flows in one benchmark, this would be pretty easy to do
* Provision separate coins so that that each flow has its own coin to use for gas (this will work, but requires more setup)
* Just use separate accounts

Reply:
Hi @hayes-mysten just to clarify, we're not running the transactions concurrently per se. The test instances are running concurrently, but the transactions are spaced out.

I think this is analogous to two instances running separately but not concurrently (e.g., a desktop wallet and a mobile wallet) and they should reasonably try to update and not step on each other's toes (in Aptos, they fetch the latest sequence number from the chain).

My understanding is the gas object optimization would not work in the above scenario, so it's clearer to remove it.

Reply:
There will still be a risk of conflict between executions, but that risk as you mentioned is based on the interval at which they are running.  None of the other networks are running multiple benchmarks off the same account though.  I've removed the gas caching for now, but it does feel a little inconsistent between networks